examples/kustomization/tenant-kes-encryption/tenant.yaml

```
@@ -7,7 +7,7 @@ spec:
   ## Define configuration for KES (stateless and distributed key-management system)
   ## Refer https://github.com/minio/kes
   kes:
-    image: "" # minio/kes:v0.18.0
+    image: "" # minio/kes:v0.22.3
```

Latest KES version https://github.com/minio/kes/releases/tag/2023-02-15T14-54-37Z
```suggestion
    image: "" # minio/kes:2023-02-15T14-54-37Z
```